pkg/operator/controller/route-status/controller_test.go

```diff
@@ -0,0 +1,197 @@
+package routestatus
+
+import (
+	"context"
+	v12 "github.com/openshift/api/operator/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	"github.com/openshift/cluster-ingress-operator/test/unit"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"testing"
+)
+
+// Test_Reconcile verifies Reconcile
+// Note: This effectively tests clearStaleRouteAdmittedStatus as well
+func Test_Reconcile(t *testing.T) {
+	reconcileRequestRoute := reconcile.Request{
+		NamespacedName: types.NamespacedName{
+			Name:      "route",
+			Namespace: "ns",
+		},
+	}
+	testCases := []struct {
+		name              string
+		request           reconcile.Request
```

Comment:
Does `request` need to be part of the test input?  It appears to be the same in every test case.